docs/UserGuide.md

```diff
@@ -178,15 +168,23 @@ Adds a company to the InternBook.
+| Phone                      | Optional            | -p   | Phone numbers should be 3 or more digits                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| Start Date                 | Optional            | -d1  | Dates should be in the form YYYY-MM-DD<br/>Start date must be earlier than end date                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| End Date                   | Optional            | -d2  | Dates should be in the form YYYY-MM-DD<br/>End date must be later than start date                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
For dates, could mention that start date can be same as end date

Reply:
made the suggested ammendments